tests/Message/CreatePaymentMethodRequestTest.php

```diff
@@ -245,9 +245,9 @@ public function testGetDataDoNotUpdateSubscriptions()
     /**
      * @return void
      */
-    public function testGetDataDeactivatePaymentMethod()
+    public function testGetDataPaymentMethodNotActive()
     {
-        $data = $this->request->setActivatePaymentMethod(false)->getData();
+        $data = $this->request->setActive(false)->getData();
```

Comment:
in `testGetData`, can you add an assertion that active is true? that'll test the default value

Reply:
I already added that check in testGetData at line 184

Reply:
ah great, i missed that!